tokens/telefonica.json

```
@@ -552,12 +572,22 @@
       "type": "color",
       "description": "telefonicaBlue70"
     },
-    "buttonSecondaryBackground": {
-      "value": "{palette.telefonicaBlue}",
+    "buttonSecondaryBackgroundSelected": {
+      "value": "{palette.telefonicaBlue70}",
       "type": "color",
-      "description": "telefonicaBlue"
+      "description": "telefonicaBlue70"
     },
-    "buttonSecondaryBackgroundSelected": {
+    "buttonSecondaryBorder": {
+      "value": "{palette.white}",
+      "type": "color",
+      "description": "white"
+    },
+    " buttonSecondaryBackgroundHover": {
```

```suggestion
    "buttonSecondaryBackgroundHover": {
```
you have a space at the beginning of this! these space cause problems, remove it!